scripts/install_dependencies.sh

```diff
@@ -40,6 +40,5 @@ while [ "$1" != "" ]; do
 done
 
 "${SC_MACHINE_PATH}/scripts/install_deps_ubuntu.sh" "${args[@]}"
```

Comment:
Don't change this script. It is used to install platform natively. If you want to use this logic in platform installation throught sc-component-manager, you can implement separate logic in this script to install platform core (f. e. `install_dependencies.sh` with flag `--core` to install only core sc-machine components) or implement another script (f. e. `install_core_dependencies.sh`)